src/behavior-considered-undefined.md

```diff
@@ -58,7 +58,19 @@ Please read the [Rustonomicon] before writing unsafe code.
 * Invoking undefined behavior via compiler intrinsics.
 * Executing code compiled with platform features that the current platform
   does not support (see [`target_feature`]), *except* if the platform explicitly documents this to be safe.
-* Calling a function with the wrong call ABI or unwinding from a function with the wrong unwind ABI.
+* Calling a function with the wrong call ABI or unwinding from a function with the wrong unwind ABI
```

Comment:
This PR significantly increases the number of top-level bullet items in the "list of UB" for Rust. That is not appropriate, as these all fall under "doing unwinding wrong". There is also significant overlap with "unwinding from a function with the wrong unwind ABI".

Is there a way to structure this that does not make one third of the UB list about unwinding?  I am thinking that most of this could become a separate section below and is just referenced up here, like we already do for some other items.